tests/nix/suite/test_basic_deploy/default.nix

```diff
@@ -0,0 +1,21 @@
+{ config, ... }:
+{
+  wire.testing.test_basic_deploy = {
+    nodes.deployer = {
+      _wire.deployer = true;
+    };
+    nodes.receiver = {
+      _wire.receiver = true;
+    };
+    testScript = ''
+      receiver.wait_for_unit("multi-user.target")
+      receiver.wait_for_unit("sshd.service")
```

Comment:
Should probably be default in the test script

Reply:
I will add a way to add to the test script via options, as I can't just yank and paste this to the testscript.